### PR TITLE
Move training into v1.1 dir and use ./cockroach everywhere

### PR DESCRIPTION
**Pull request by jseldess**

### CODE REVIEW

**Comment by a-robinson** [comment]
:lgtm: but the real test for this is running through everything again to make sure nothing got missed. Is anyone going to be doing that today? I don't know what motivated the move to the v1.1 directory.

---

Reviewed 88 of 88 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-:-L5EMbTLdo0evLkDClU-:b-suce8z)*

**Comment by petermattis** [comment]
I’ll run through all of the Wednesday material today after this is merged.

**Comment by jseldess** [comment]
Great. I'll run through everything as well, and I'll try to get someone else on docs to do so, too. 

I moved to the v1.1 directory so it's possible to version this content. Soon after the mesos training, I'll create a version in the v2.0 directory and focus on that going forward.

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-:-L5EdWZ-o0xIgu-yFQUy:bv6ao19)*

**Comment by jseldess** [comment]
*[v1.1/training/cluster-startup-and-scaling.md, line 55 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5EIE4b21Lf5O-E52Lc:-L5Ef-FAZxH4pZ970Hl0:bqihl1j) ([raw file](https://github.com/cockroachdb/docs/blob/1f8449b757d1e966f40480d7b529f35fa6988e66/v1.1/training/cluster-startup-and-scaling.md#L55)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

I'm guessing you've borrowed these instructions from elsewhere. Do you know why we use `wget` on linux and `curl` on os x?
</blockquote></details>

I think Sean made that decision early on, but it might've been me? What would you prefer/expect instead? From what I remember, `curl` doesn't work out of the box on linux and `wget` doesn't work out of the box on OS X?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by jseldess** [comment]
*[v1.1/training/fault-tolerance-and-automated-repair.md, line 94 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5EIn27Awa6tck5dQ2N:-L5EfQmYd0_tmqvtt52t:bhy9ot6) ([raw file](https://github.com/cockroachdb/docs/blob/1f8449b757d1e966f40480d7b529f35fa6988e66/v1.1/training/fault-tolerance-and-automated-repair.md#L94)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

Let's verify these commands work after merging.
</blockquote></details>

Will do.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by petermattis** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[v1.1/training/cluster-startup-and-scaling.md, line 55 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5EIE4b21Lf5O-E52Lc:-L5Ei8t4uSJS8Mln41rH:bhusoi3) ([raw file](https://github.com/cockroachdb/docs/blob/1f8449b757d1e966f40480d7b529f35fa6988e66/v1.1/training/cluster-startup-and-scaling.md#L55)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

I think Sean made that decision early on, but it might've been me? What would you prefer/expect instead? From what I remember, `curl` doesn't work out of the box on linux and `wget` doesn't work out of the box on OS X?
</blockquote></details>

Naively I would have expected the same command to work on both. Let's not change this now as I imagine you (or Sean) previously tested what works.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by jseldess** [comment]
*[v1.1/training/cluster-startup-and-scaling.md, line 55 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5EIE4b21Lf5O-E52Lc:-L5Ek8ySWaFw5TbQCc19:b2nhktp) ([raw file](https://github.com/cockroachdb/docs/blob/1f8449b757d1e966f40480d7b529f35fa6988e66/v1.1/training/cluster-startup-and-scaling.md#L55)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

Naively I would have expected the same command to work on both. Let's not change this now as I imagine you (or Sean) previously tested what works.
</blockquote></details>

OK.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by jseldess** [comment]
@a-robinson, apologies for overloading this PR, but can you take a look at the cluster upgrades lab and slides as well?

**Comment by petermattis** [comment]
Review status: 28 of 46 files reviewed at latest revision, 3 unresolved discussions, some commit checks pending.

---

*[v1.1/training/cluster-startup-and-scaling.md, line 65 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FBU-MyLZ6kh5kpxVE:-L5FBU-MyLZ6kh5kpxVF:b-7a410e) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L65)):*
> ```Markdown
>     ~~~ shell
>     $ mv cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach . \
>     | rm -rf cockroach-{{ page.release_info.version }}.darwin-10.9-amd64
> ```

You want a semicolon between the `mv` and `rm` commands, not a pipe. The pipe allows both commands to run concurrently and can race causing the `mv` to fail. The semicolon runs them in sequence. Or perhaps even better is to use `mv ... && rm ...` (i.e. only run the `rm` if the `mv` succeeded). This applies both here and for the `linux` case below.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by petermattis** [comment]
Review status: 28 of 46 files reviewed at latest revision, 3 unresolved discussions, some commit checks pending.

---

*[v1.1/training/cluster-startup-and-scaling.md, line 65 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FBU-MyLZ6kh5kpxVE:-L5FBsNk2Hw88c2CX_ew:b-vighlg) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L65)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

You want a semicolon between the `mv` and `rm` commands, not a pipe. The pipe allows both commands to run concurrently and can race causing the `mv` to fail. The semicolon runs them in sequence. Or perhaps even better is to use `mv ... && rm ...` (i.e. only run the `rm` if the `mv` succeeded). This applies both here and for the `linux` case below.
</blockquote></details>

Alternately, perhaps we should create a symlink via `ln -sf cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach .`.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by petermattis** [comment]
Worked through the `Cluster Startup and Scaling` lab with one small hiccup. 

---

Review status: 28 of 46 files reviewed at latest revision, 5 unresolved discussions, some commit checks pending.

---

*[v1.1/training/cluster-startup-and-scaling.md, line 173 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FCNAveV6Jj6YDg_Cj:-L5FCNAveV6Jj6YDg_Ck:bib50sx) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L173)):*
> ```Markdown
>     ~~~
>     CockroachDB node starting at 2018-02-02 05:08:22.841392 +0000 UTC
>     build:      CCL v1.1.4 @ 2018/01/08 17:30:06 (go1.8.3)
> ```

`s/v1.1.4/v1.1.5/g

---

*[v1.1/training/cluster-startup-and-scaling.md, line 207 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FCS0dek1a7GOUe2xW:-L5FCS0dek1a7GOUe2xX:b-nkqtt) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L207)):*
> ```Markdown
>     | id |     address     | build  |     updated_at      |     started_at      |
>     +----+-----------------+--------+---------------------+---------------------+
>     |  1 | localhost:26257 | v1.1.4 | 2018-02-02 16:48:27 | 2018-02-02 16:48:17 |
> ```

`s/v1.1.4/v1.1.5/g`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-:-L5FCoeYJ3gBATuRLYN_:b-x18gxu)*

**Comment by jseldess** [comment]
*[v1.1/training/cluster-startup-and-scaling.md, line 65 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FBU-MyLZ6kh5kpxVE:-L5FDhwWBSRgZdZZLSEY:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L65)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

Alternately, perhaps we should create a symlink via `ln -sf cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach .`.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by jseldess** [comment]
*[v1.1/training/cluster-startup-and-scaling.md, line 173 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FCNAveV6Jj6YDg_Cj:-L5FF21rpAEYtxYvg3VK:b-2oakty) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L173)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

`s/v1.1.4/v1.1.5/g
</blockquote></details>

Change to using a variable here and elsewhere.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by jseldess** [comment]
*[v1.1/training/cluster-startup-and-scaling.md, line 207 at r3](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FCS0dek1a7GOUe2xW:-L5FF8xqAl37Nb0NlTWM:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/42e37a3b5a6764669af1dc006f6e2ecb85013d35/v1.1/training/cluster-startup-and-scaling.md#L207)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

`s/v1.1.4/v1.1.5/g`
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by jseldess** [comment]
*[v1.1/training/security.md, line 53 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5EJ1Dt8teupMYfF8ZB:-L5EgRG6Xpo-sZGN_F_e:b-9bhgil) ([raw file](https://github.com/cockroachdb/docs/blob/1f8449b757d1e966f40480d7b529f35fa6988e66/v1.1/training/security.md#L53)):*
<details><summary><i>Previously, petermattis (Peter Mattis) wrote…</i></summary><blockquote>

There are a handful of shell commands that appear to have tabs at the beginnings of the line. Is that intentional?
</blockquote></details>

Hm, not intentional. I've changed to spaces. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by petermattis** [comment]
Review status: 27 of 46 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[v1.1/training/fault-tolerance-and-automated-repair.md, line 29 at r8](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5FDa1oJVKnzyrcW48Q:-L5FDa1oJVKnzyrcW48R:b-wg7fio) ([raw file](https://github.com/cockroachdb/docs/blob/80ab25b536584c746af20733a75efe81a9d21aa4/v1.1/training/fault-tolerance-and-automated-repair.md#L29)):*
> ```Markdown
>     {% include copy-clipboard.html %}
>     ~~~ shell
>     $ brew install haproxy
> ```

Should we track down instructions for linux? Seems irritatingly difficult given the number of possible package managers out there, but perhaps we can have the command for Ubuntu.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

File: v1.1/training/cluster-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+Once all nodes are on the same upgraded version, update the `version` cluster setting to enable certain performance improvements and bug fixes that were introduced in v2.0:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ ./cockroach sql \
+--insecure \
+--execute="SET CLUSTER SETTING version = crdb_internal.node_executable_version();"
+~~~
+
+{{site.data.alerts.callout_info}}This final step is required after upgrading from v1.1.x to v2.0. For upgrades within the 2.0.x series, you don't need to take any further action.{{site.data.alerts.end}}
```
Comment (a-robinson) [inline v1.1/training/cluster-upgrade.md]: Shouldn't we also warn that this eliminates the ability to downgrade? We could rephrase the intro to this step as "update the `version` cluster setting to enable certain backwards-incompatible performance improvements and bug fixes that were introduced in v2.0:"

**Comment by jseldess** [comment]
*[v1.1/training/cluster-upgrade.md, line 270 at r10](https://reviewable.io:443/reviews/cockroachdb/docs/2493#-L5Dcym7hCZ5CTHWFsZ--r10-270:-L5FkKWibWT0w9V-0-KC:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/7cf2dd9c6fec0502ffe23811e5082403b344b2f5/v1.1/training/cluster-upgrade.md#L270)):*
<details><summary><i>Previously, a-robinson (Alex Robinson) wrote…</i></summary><blockquote>

Shouldn't we also warn that this eliminates the ability to downgrade? We could rephrase the intro to this step as "update the `version` cluster setting to enable certain backwards-incompatible performance improvements and bug fixes that were introduced in v2.0:"
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2493)*

**Comment by petermattis** [comment]
Per Alex's comment on #2495, I think it is worthwhile to merge this soon as the renaming to the v1.1 directory will cause headaches for other PRs.

**Comment by a-robinson** [comment]
Yeah, in retrospect it would have been nice to get the v1.1 move in as soon as was possible and done the rest separately. But we should definitely get this in soon so that we have time to rebase and retest other PRs.

**Comment by jseldess** [comment]
Sorry for the delay on this! I'm merging now. If there are problems, I'll deal with them.

**Pull request merged**